src/OrchardCore.Modules/OrchardCore.OpenId/Views/OpenIdValidationSettings.Edit.cshtml

```diff
@@ -28,6 +28,13 @@
     <span class="hint">@T["The address of the remote OpenID Connect server. Using a HTTPS address is strongly recommended."]</span>
 </div>
 
+<div class="mb-3" asp-validation-class-for="MetadataAddress">
+    <label asp-for="MetadataAddress">@T["Well Known Metadata Address"]</label>
```

Comment:
`Well Known Metadata Address` may not be the best choice: the .well-known endpoints are not necessarily related to OIDC and you have tons of other standard endpoints for many other scenarios.

Maybe simply `Metadata Address`? or `OpenID Connect Metadata Address`?

Reply:
We don't use them right now, we assume strings can be null.

Reply:
Updated